client/src/components/dataset/sidebars/GenerateGeopoints.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Immutable from 'immutable';
+import _ from 'lodash';
```

Comment:
I thought that using lodash here might bloat the bundle size, and I was going to suggest that perhaps it's overkill give that we're only using it to capitalize some strings - but in fact lodash is being loaded anyway in the `app` bundle, so using it here has zero effect on the size of the `Dataset` bundle. So this comment can be safely ignored :)

Reply:
Yeah, I figured it would have no impact here ^^